backend/fms_core/models/sample.py

```diff
@@ -117,12 +117,19 @@ class Sample(TrackedModel):
     sample_kind = models.ForeignKey(SampleKind, on_delete=models.PROTECT,
                                     help_text="Biological material collected from study subject "
                                               "during the conduct of a genomic study project.")
-    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH, validators=[name_validator],
-                            help_text="Sample name.")
+
+    name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH,
+                            validators=[name_validator],
+                            help_text="Sample name.",
+                            error_messages={'blank': 'Sample name cannot be blank'})
```

Comment:
Is it necessary to override the very similar default error message for a blank value?

Reply:
This particular error message had to be overridden because the default message would be "Name - This field cannot be blank." This ticket: https://206.12.92.46/issues/334 states that it might be confused with container name.

Reply:
This might be useful since by default it returns "name cannot be blank" which is not so useful when you have container name, individual name, sample name ... maybe though we could find a more generic approach to this by overloading the models default error_messages to prepend the name of the model as part of the message (%(app_label)s_%(class)s). Not sure of realistic this is though.